src/engine.js

```diff
@@ -139,8 +140,8 @@ Engine.prototype.reload = function (options) {
       resolve();
     };
     options.error = function (serverResponse) {
-      oldError(serverResponse);
-      reject(serverResponse);
+      var parsedError = oldError(serverResponse);
```

Comment:
whats `oldError`  ?

Reply:
`oldXXX` is the name that was already there assigned to the error and success callbacks.